apps/src/bin/anoma-wallet/cli.rs

```diff
@@ -190,17 +190,36 @@ fn address_list(ctx: Context) {
     }
 }
 
-/// Find address by its alias.
-fn address_find(ctx: Context, args: args::AddressFind) {
+/// Find address (alias) by its alias (address).
+fn address_or_alias_find(ctx: Context, args: args::AddressOrAliasFind) {
     let wallet = ctx.wallet;
-    if let Some(address) = wallet.find_address(&args.alias) {
-        println!("Found address {}", address.to_pretty_string());
-    } else {
+    if args.address.is_some() && args.alias.is_some() {
         println!(
```

Comment:
let's make it fail here
```suggestion
        assert!(false,
```